proto/auth.proto

```diff
@@ -69,8 +69,34 @@ message _GenerateApiTokenRequest {
         Expires expires = 2;
     }
 
-    // session token recieved from `momento login` command
-    string session_token = 3;
+    enum Role {
+      ReadWrite = 0;
```

Comment:
🤔 this is going to be a default value deserialized for lots of stuff. Since this goes in the restrictions block I think you want a `PermitNone = 0` and a `ReadWrite = 1`?

Reply:
ah yea good point